Selenium.Axe/Selenium.Axe/HtmlReport.cs

```diff
@@ -425,8 +426,59 @@ private static void GetReadableAxeResults(AxeResultItem[] results, string type,
 
                     htmlAndSelector.InnerHtml = content.ToString();
                     htmlAndSelectorWrapper.AppendChild(htmlAndSelector);
+
+                    var anyCheckResults = item.Any;
+                    var allCheckResults = item.All;
+                    var noneCheckResults = item.None;
+
+                    int checkResultsCount = anyCheckResults.Length + allCheckResults.Length + noneCheckResults.Length;
+
+                    // Add fixes if this is for violations
+                    if (ResultType.Violations.Equals(type) && checkResultsCount > 0)
+                    {
+                        if (checkResultsCount == 1)
+                        {
+                            htmlAndSelector = doc.CreateTextNode("To fix this violation address the following issue:");
+                        }
+                        else
+                        {
+                            htmlAndSelector = doc.CreateTextNode("To fix this violation address at least one of the following issues:");
+                        }
+
+                        htmlAndSelectorWrapper.AppendChild(htmlAndSelector);
+
+                        htmlAndSelector = doc.CreateElement("p");
+                        htmlAndSelector.SetAttributeValue("class", "wrapOne");
+                        htmlAndSelector.InnerHtml = $"{HttpUtility.HtmlEncode(item.Html)}";
+
+                        content = new StringBuilder();
+                        htmlAndSelector = doc.CreateElement("p");
+                        htmlAndSelector.SetAttributeValue("class", "wrapTwo");
+
+                        content.AppendLine("<ul>");
+
+                        foreach (var checkResult in anyCheckResults)
+                        {
+                            content.AppendLine($"<li>{HttpUtility.HtmlEncode(checkResult.Impact.ToUpper())}: {HttpUtility.HtmlEncode(checkResult.Message)}</li>");
+                        }
+
+                        foreach (var checkResult in allCheckResults)
```

Comment:
I think this is going to present instructions that are a little bit off in the event that there are multiple all/none results; the text will say "to fix this violation address at least one of the following issues", but the correct semantics are really "fix every "all" issue, every "none" issue, and at least one "any" issue".

Reply:
Updated